### PR TITLE
Feat/cow widget

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

<<if there's an issue>>Fixes #issueNumber

*High-level description of what your changes are accomplishing*

*Add screenshots if applicable. Images are nice :)*

  # To Test

1. <<Step one>> Open the page `about`
- [ ] <<What to expect?>> Verify it contains about information...
- [ ] Checkbox Style list of things a QA person could verify, i.e.
- [ ] Should display Text Input our storybook
- [ ] Input should not accept Numbers
2. <<Step two>> ...

  # Background

*Optional: Give background information for changes you've made, that might be difficult to explain via comments*

### CODE REVIEW

File: src/modules/wallet/web3-react/connectors/Injected/IFrameEthereumProvider.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { IFrameEthereumProvider as OriginalIFrameEthereumProvider } from '@ethvault/iframe-provider'
+
+export class IFrameEthereumProvider extends OriginalIFrameEthereumProvider {
```
Comment (anxolin) [inline src/modules/wallet/web3-react/connectors/Injected/IFrameEthereumProvider.ts:3]: for now you just extended, so we can add stuff, but we just delegate, right? in principle we could even use the original? (happy to leave it extended, just curious)

File: src/modules/wallet/web3-react/connectors/Injected/IFrameEthereumProvider.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  request<T>({ method, params }: { method: string; params?: any[] }) {
+    return this.send<typeof params, T>(method, params)
+  }
+
+  isConnected(): true {
+    return true
```
Comment (anxolin) [inline src/modules/wallet/web3-react/connectors/Injected/IFrameEthereumProvider.ts:9]: why is this is true always?

**Review by anxolin — APPROVED** [review]
Nice!

File: src/modules/wallet/web3-react/connectors/Injected/IFrameEthereumProvider.ts (hunk 1 above)
Comment (shoom3301) [reply src/modules/wallet/web3-react/connectors/Injected/IFrameEthereumProvider.ts:3]: Unfortunately, the original is not 100% relevant, it doesn't implement required `request` method

**Pull request closed without merging**